src/redux/actions/actionCreator.ts

```diff
@@ -0,0 +1,16 @@
+import { Category } from "../../types/Category";
+import actionTypes from "./actionTypes";
+
+export const loadCategoriesAction = (categories: Category[]) => ({
```

Comment:
Future: should be renamed to something more meaningful such as : `createLoadCategoriesAction` or `buildLoadCategoriesAction`